### ISSUE TITLE
[ENHANCEMENT] Add install mode build CI release

### ISSUE THREAD

**Issue by guangbochen** [harvester/harvester#4211]
**Is your enhancement related to a problem? Please describe.**
This is a follow-up issue of https://github.com/harvester/harvester/issues/2198. It was planned only to release the install qcow image on tag or daily cron build.

**Describe the solution you'd like**
Enable `BUILD_QCOW=true` on the daily cron job or tag build.

However, this is blocked by the CI of missing KVM kernel support https://drone-publish.rancher.io/harvester/harvester-installer/1524/1/2
``
Could not access KVM kernel module: No such file or directory
qemu-system-x86_64: failed to initialize kvm: No such file or directory
``

**Describe alternatives you've considered**

**Additional context**

**Comment by bk201** [comment]
We can see if this can be achieved with github action migration.

**Comment by bk201** [comment]
Need to check if the new GH action runner has kvm capability.

**Comment by innobead** [comment]
> Need to check if the new GH action runner has kvm capability.

Our new runners are VM based, so I assume this should not be an issue anymore?
